RELEASE.md

```diff
@@ -1,6 +1,16 @@
 # Automated releases
 TBD
 
+# Testing installer builds of any netplugin branch
+To create a full installer for legacy docker swarm of any branch of contiv, additionally set these two environment variables:
+
+* NETPLUGIN_BRANCH - the branch to compile instead of latest release
```

Comment:
Isn't this "branch to compile instead of current branch"?

Reply:
if this is not set, install pulls release versions only based on a checked in variable value or environment override, which is unrelated to any workspace branch

Reply:
Could the description be something more like that?  The current text didn't imply any of that to me as someone reading it for the first time

Reply:
done